### PR TITLE
JOSS paper

### PR DESCRIPTION
**Pull request by tupui**
Doc for JOSS: https://joss.readthedocs.io/en/latest/submitting.html#example-paper-and-bibliography

### CODE REVIEW

**Comment by tupui** [comment]
The structure is there, now "just" need to do some writing 😅 It's a one page thing, I should be able to do that (and could help with what needs to end up in the book??)

Only the md and bib files need to be edited. To see how it renders, push on the branch and you can look at the automatic action `Paper Draft (push)`:
Screenshot 2023-10-31 at 23 42 03

Then click summary and download the artifact:

Screenshot 2023-10-31 at 23 42 50

We cannot change anything to the styling, everything is done automagically and there is no tuning allowed.

cc @gnopik

**Comment by tupui** [comment]
@gnopik I pushed a quick update. Besides the info there that I need to rework slightly, is there anything special you want to see here? I guess at least 1-2 citations for SimDec itself, something else?

File: joss/paper.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+available on PyPi, SimDec is also made available
+to practitioners through an online dashboard at https://simdec.io. The
+innovative methods of SimDec, the project relies on powerful methods
+from SALib [@Herman2017] and SciPy [@Virtanen2020; @Roy2023]-notably the
+QMC capabilities with `sp.stats.qmc` and sensitivity indices with
+`sp.stats.sensitivity_indices`. The dashboard is made possible thanks to Panel.
+
+# Acknowledgements
+
+The authors thank ...
```
Comment (gnopik) [inline joss/paper.md:71]: The work on this open-source software was supported by grant #220177 from Finnish Foundation for Economic Foundation

File: joss/paper.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    - name: LUT Business School, LUT University, Lappeenranta, Finland
+      index: 2
+date: 1 April 2024
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Uncertainties are everywhere! Whether you are developing a new AI system,
```
Comment (gnopik) [inline joss/paper.md:29]: Suggestion to use "." instead of "!"

**Review by gnopik — CHANGES_REQUESTED** [review]
All good, see two minor comments and the addition to the acknowledgments.

File: joss/paper.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+uncertainties play. Thanks to a clear Python Application Programming Interface (API) and an interactive web
+dashboard, making uncertainty analysis accessible to everyone.
+
+# Statement of need
+
+From real life experiments to numerical simulations, uncertainties play a
+crucial role in the system under study. With the advent of Artificial
+Intelligence and new regulations such as the [AI Act](https://artificialintelligenceact.eu) or the
+*Better Regulation Guideline* [@europeancommission2021], there is a growing need for explainability and
+impact assessments of systems under uncertainties.
```
Comment (tupui) [inline joss/paper.md:44]:
```suggestion
From real life experiments to numerical simulations, uncertainties play a
crucial role in the system under study. With the advent of Artificial
Intelligence and new regulations such the [AI Act](https://artificialintelligenceact.eu) or the
*Better Regulation Guideline* [@europeancommission2021], there is a growing need for explainability and
impact assessments of systems under uncertainties.
```

File: joss/paper.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Uncertainties are everywhere. Whether you are developing a new Artificial Intelligence (AI) system,
+running complex simulations or making an experiment in a lab, uncertainties
+influence the system. And you need a way to understand how these impact your
+results.
```
Comment (gnopik) [inline joss/paper.md:32]:
```suggestion
influence the system. Therefore, an approach is needed to understand how these uncertainties impact the system's performance.
```

File: joss/paper.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+SimDec offers a novel visual way to understand the intricate role that
+uncertainties play. Thanks to a clear Python Application Programming Interface (API) and an interactive web
+dashboard, making uncertainty analysis accessible to everyone.
```
Comment (gnopik) [inline joss/paper.md:36]:
```suggestion
uncertainties play. A clear Python Application Programming Interface (API) and a no-code interactive web
dashboard make uncertainty analysis accessible to everyone.
```

File: joss/paper.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+Traditional methods to analyse the uncertainties focus on quantitative methods
+to compare the importance of factors, there is a large body of literature and
+the field is known as: Sensitivity Analysis (SA) [@Saltelli2007]. The indices of Sobol' are a
+prominent example of such methods [@sobol1993].
+
+Simulation Decomposition is a hybrid uncertainty-sensitivity analysis approach
+that reveals the critical behavior of a computational model or an empirical
+dataset [@Kozlova2024]. It decomposes the distribution of the output
+(target variable) by the multivariable scenarios, formed out of the most
+influential input variables. The resulting visualization shows how different
```
Comment (gnopik) [inline joss/paper.md:55]:
```suggestion
dataset [@Kozlova2024]. It decomposes the distribution of the output
(target variable) by automatically forming scenarios that reveal the most critical behavior of the system. The scenarios are formed out of the most influential input variables (defined with variance-based sensitivity indices) by breaking down their numeric ranges into states (e.g. low and high) and creating an exhaustive list of their combinations (e.g. (i) low A & low B, (ii) low A & high B, (iii) high A & low B, and (iv) high A and high B). The resulting visualization shows how different
```

File: joss/paper.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+output ranges can be achieved and what kind of critical interactions affect
+the output–as seen in \autoref{fig:simdec}. The method has shown value for
+various computational models from different fields, including business,
+environment, and engineering, as well as an emerging evidence of use for
+empirical data and AI.
+
+![SimDec: explanation of output by most important inputs.\label{fig:simdec}](simdec_presentation.png)
```
Comment (gnopik) [inline joss/paper.md:62]:
```suggestion
![SimDec: explanation of output by most important inputs. A simulation dataset of a structural reliability model with one key output variable and four input variables is used for this case. Inputs 3 and 1 have the highest sensitivity indices and thus are automatically chosen for decomposition. The most influential input 3 divides the distribution of the output into three main states with distinct colors. Input 1 further subdivides them into shades. From the graph, it becomes obvious that input 1 influences the output when input 3 is low, but has a negligible effect when input 3 is medium or high.\label{fig:simdec}](simdec_presentation.png)
```

File: joss/paper.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+Besides proposing a comprehensive yet simple API through a Python package
+available on PyPi, SimDec is also made available
+to practitioners through an online dashboard at [https://simdec.io](https://simdec.io). The project
+relies on powerful methods from SALib [@Herman2017] and
```
Comment (gnopik) [inline joss/paper.md:67]:
```suggestion
relies on powerful variance-based sensitivity analysis methods from SALib [@Herman2017] and
```

File: joss/paper.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+impact assessments of systems under uncertainties.
+
+Traditional methods to analyse the uncertainties focus on quantitative methods
+to compare the importance of factors, there is a large body of literature and
+the field is known as: Sensitivity Analysis (SA) [@Saltelli2007]. The indices of Sobol' are a
+prominent example of such methods [@sobol1993].
+
+Simulation Decomposition is a hybrid uncertainty-sensitivity analysis approach
+that reveals the critical behavior of a computational model or an empirical
+dataset [@Kozlova2024]. It decomposes the distribution of the output
```
Comment (gnopik) [inline joss/paper.md:53]:
```suggestion
Simulation Decomposition or SimDec moves the field of SA forward by supplementing the computation of sensitivity indices with the visualization of the type of interactions involved, which proves critical for understanding the system's behavior and decision-making [@Kozlova2024]. 
In short, SimDec is a hybrid uncertainty-sensitivity analysis approach
that reveals the critical behavior of a computational model or an empirical
dataset. It decomposes the distribution of the output
```

File: joss/paper.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+In short, SimDec is a hybrid uncertainty-sensitivity analysis approach
+that reveals the critical behavior of a computational model or an empirical
+dataset. It decomposes the distribution of the output
+(target variable) by automatically forming scenarios that reveal the most critical behavior of the system. The scenarios are formed out of the most influential input variables (defined with variance-based sensitivity indices) by breaking down their numeric ranges into states (e.g. _low_ and _high_) and creating an exhaustive list of their combinations (e.g. (i) _low_ _**A**_ & _low_ _**B**_, (ii) _low_ _**A**_ & _high_ _**B**_, (iii) _high_ **_A_** & _low_ **_B_**, and (iv) _high_ **_A_** and _high_ **_B_**). The resulting visualization shows how different
+output ranges can be achieved and what kind of critical interactions affect
+the output–as seen in \autoref{fig:simdec}. The method has shown value for
```
Comment (matt-graham) [inline joss/paper.md:56]:
```suggestion
the output—as seen in \autoref{fig:simdec}. The method has shown value for
```

File: joss/paper.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+SimDec offers a novel visual way to understand the intricate role that
+uncertainties play. A clear Python Application Programming Interface (API) and a no-code interactive web
+dashboard make uncertainty analysis with SimDec accessible to everyone.
+
+# Statement of need
+
+From real life experiments to numerical simulations, uncertainties play a
+crucial role in the system under study. With the advent of Artificial
+Intelligence and new regulations such as the [AI Act](https://artificialintelligenceact.eu) or the
```
Comment (matt-graham) [inline joss/paper.md:41]:
```suggestion
crucial role in the system under study. With the advent of AI
and new regulations such as the [AI Act](https://artificialintelligenceact.eu) or the
```
As acronym already defined above

**Comment by tupui** [comment]
@gnopik I added notebook support. See https://simdec--23.org.readthedocs.build/en/23/notebooks/structural_reliability.html

**Comment by tupui** [comment]
Thank you @matt-graham, done.

**Comment by tupui** [comment]
Alright, the paper is approved 🎉 Getting this in and cutting a release now for the process.

**Pull request merged**